### PR TITLE
Fixes and improvements in the CoC section

### PR DESCRIPTION
**Pull request by ecamellini**

### CODE REVIEW

**Review by gabro — APPROVED** [review]
Cool! Thanks for the fixes!

File: book/CoC.md [hunk 1]
```diff
@@ -1,43 +1,38 @@
...
+There were many points discussed, but we picked the top 5 so that we can really commit to them. This is exactly in the spirit of the second point of the CoC itself (recursion is a bitch).
 
 ![](code_of_conduct.png)
-Like everything in buildo, nothing is set in stone. CoC reflects  values we believe in. If we change our minds as a team, we'll change this.
+Like everything in buildo, nothing is set in stone. This CoC reflects the values we believe in; if we change our minds as a team we'll adapt it and everyone is more than welcome to propose changes.
 
-Everyone is more than welcome to propose changes to the CoC.
-
-In the next sections we'll give suggestions on how to follow the CoC with the help of tools.
+In the next subsections, we'll give suggestions on how to follow the CoC with the help of tools.
```
Comment (bytecodeguru) [inline book/CoC.md]: Toglierei la virgola o rifraserei mettendo il soggetto all'inizio.

Comment (ecamellini) [reply book/CoC.md]: 👍 ci stava togliere al virgola

File: book/CoC.md [hunk 2]
```diff
@@ -1,43 +1,38 @@
...
 - Requester: the person who requested the task, and who accepts it when completed.
 - Performer: the person who's responsible for the execution of the task, and who notifies the requester when done.
 - Task: a clear specification of all that should be done.
-- Due date: a date by which the task will be completed.
+- Due date: a date by which the task should be completed.
 
 Consider this a negotiation, not an order. Both performer and requester must agree on the commitment.
 
-Remember to notify the other if things change. In particular if something happens and the commitment cannot be hold.
+Remember to notify the other people involved if things change; in particular, if something happens and the commitment cannot be held.
```
Comment (bytecodeguru) [inline book/CoC.md]: in particular ... -> especially if the commitment cannot be held.

File: book/CoC.md [hunk 3]
```diff
@@ -66,10 +59,10 @@ Organize meetings after 10am and before 6pm to make sure everyone attends, and t
 
 ## Prefer small things that you complete to big, long lasting tasks
 
-This is a general principle, and it helps a lot. You can see it in effect with GitHub issues (we keep them small), iteration planning, size of features in development, etc.
+This is a general principle, and it helps a lot. You will see it in effect in our GitHub issues (we keep them small), our iteration planning, the size of features in development, etc.
 
-A good way to see if your really doing small things, and keeping the "flow", is to count your WIP (work in progress). We do this in GitHub with the WIP label, but you can do it with everything you do. The goal is to keep low WIP and high throughput. This is *flow*.
+A good way to see if your really doing small things, and keeping the "flow", is to count your WIP (work in progress) taks. We do this in GitHub with the WIP label, but you can do it with everything you are working on. The goal is to keep low WIP and high throughput. This is *flow*.
```
Comment (bytecodeguru) [inline book/CoC.md]: your -> you're

**Review by bytecodeguru — APPROVED** [review]
Just some minor fixes. 💯  overall.

File: book/CoC.md (hunk 2 above)
Comment (ecamellini) [reply book/CoC.md]: 👍

File: book/CoC.md (hunk 3 above)
Comment (ecamellini) [reply book/CoC.md]: 👍

**Pull request merged**